src/hooks/folderPageHooks.js

```diff
@@ -0,0 +1,41 @@
+import { useState, useEffect } from "react";
+import axios from "axios";
+import { FOLDER_PAGE_API_URL } from "../constants/constants";
+
+export function useFolderPageUser(){
+  const [user, setUser] = useState();
+
+  const getUser = async() => {
+    try{
+      const response = await axios.get(`${FOLDER_PAGE_API_URL}/1`);
+      setUser(response.data.data[0]);
```

Comment:
1번째 유저가 왜 FolderPageUser인지 드러나면 좋겠습니다. 이번 미션에서의 한정되는 요구사항 이라서 [0]을 쓰시긴 했지만... 일단 변수명으로라도요

+이름을 드러내려면 then을 쓰면 편할거같네요

```suggestion
      const 뭔가특별한기준이되는첫번째유저 = await axios.get(`${FOLDER_PAGE_API_URL}/1`).then(res=> res.data.data[0];
      setUser(뭔가특별한기준이되는첫번째유저);
```